integration-tests/tests/test_end_to_end.tavern.yaml

```diff
@@ -23,6 +23,9 @@ marks:
       vals:
         - [psa, p1, good, full, ec.p256, good]
         - [psa, p1, good, mini, ec.p256, good]
+        - [psa, p1, missingclaims, mini, ec.p256, noident]
+        - [psa, p1, good, mini, bad, badcrypto]
+        - [psa, p1, badinstance, full, ec.p256, badinstance]
```

Comment:
ditto: why 99? If the identity is not known (i.e., the trust anchor lookup fails) shouldn't we return instance-identity=1 or 97?